### ISSUE TITLE
Quality of Life (QoL) improvements

### ISSUE THREAD

**Issue by matteo5c** [mattbona/lovely-autoencoder#2]
Usually your code is organized in two parts: 
* the source (usually in a directory called src) where you put every file with your classes and functions
* the actual scripts that you use for creating instances of your problem and then solving it

In practice: 
- [x]  write a file that contains the autoencoder and the training loop but incorporate it in a function called something like `train_ae` that takes as argument the parameters that specify the architecture and the training, such as dimension of the network layer, number of epochs, learning rate
- [x] once you built you "library", choose your favourite scripting language that launces the function `train_ae` with specified set of paramerers. I used straight up bash for a long time, now I tend to use the same language of the library. If you use bash it should be very easy to control parallel executions of `train_ae` with different parameters, but less easy to manage optional arguments that say python itself. 
- [x] store everything your code produces inside a directory called something like `results`. Make subdirectories with smart names that allow you to remember which values of the parameters were used in each experiment. You should end up with awful but very telling names like `data_nc_S_NORM8.0E-02_fixB_custom_L0.0E+00_d0.00E+00_r0.2_M7.0E+02_R1_lr-1.0E-03_A0.0E+00_0.0E+00_G0.0E+00_B-4.0E-01`

Additional QoL:
- [x] this readme is obviously useless. You shou write at least 1) what is the purpose of this project and how the data is supposed to be read 2) how the code is organized and how to install and run it 3) what do you do with the results that come out from a run: you do a plot? You compute something? Include in the project those scripts too
- [x] learn markdown (it takes like 15 minutes)
- [x] do NOT create plots during the training, you are just slowing down the training and creating a lot of junk files

**Comment by mattbona** [comment]
Thank you Matteo again for your precious advices. I've tried to improve the code according also to your suggestions! 

It would be awesome if you let me know if there are other things that you would improve or do in a different ways.
![polite-cat](https://user-images.githubusercontent.com/61428922/96745048-fa138380-13c5-11eb-925b-5f62fc1f79a2.jpg)
